rust/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN cargo new numaflow-models
 COPY ./numaflow-models/Cargo.toml ./numaflow-models/Cargo.toml
 
 RUN cargo new monovertex
-COPY monovertex/Cargo.toml ./monovertex/Cargo.toml
+COPY numaflow-core/Cargo.toml ./monovertex/Cargo.toml
```

Comment:
why copying?

Reply:
i was planning to handle that in a different PR, the reason being this is breaking and I do not want to linger this PR too long.